articles/intro-to-C++-part01/index.md

```diff
@@ -0,0 +1,136 @@
+
+Some of the best applications built with C++ are:
+
+- Apple OS X Operating System
+- Microsoft Operating System including Microsoft Office, Internet Explorer, and Visual Studio
+- MySQL Server
+- Mozilla Firefox
+- Adobe System
+- Google Chrome
+- PayPal
+- Amazon
+
+And the list will not stop. You can count for days the applications built using C++.
+
+C++ is a beginner-friendly language that anybody can start learning with or without any prior programming experience. The online community is so strong that you will never feel left out of resources and help in anything regarding C++.
+
+In this series of guides, will we go from basics to advanced level. So let's jump straight into learning.
+
+### What do you need beforehand
+- A working PC or a Computer
+- An IDE
+
+The first question that comes to your mind would be, what is an IDE? It is an integrated development environment where you will write your C++ code and run it.
+To get started, you can download [Visual studio code](https://code.visualstudio.com/) as it is simple and beginner-friendly.
+
+So let's dive into coding:
+
+### Hello World
+Let's write a Hello World program. We will break it down line by line:
+
+1. // This is a sample C++ Program
+2. #include<iostream>
+3. using namespace std;
+4. int main() {
+5. cout<<"Hello World"<<endl;
+6. return 0;
+7. }
+
+
+This program will print Hello World to the screen. Let's examine it line by line to see what it means.
+
+- Line 1 - **//This is a sample C++ Program**
+
+Any line of a program that begins with a "//" is a comment. The compiler will ignore this line.
+
+Comments are there to be read by people. You should always write comments for each major step of the algorithm to explain something weird or cool that is happening. So, anyone working with you or for you would be able to understand what you were trying to do.
+
+- Line 2 - **#include<iostream>**
+
+C++, by itself, cannot do input or output, so we add functionality to C++ by including libraries. For example, :<iostream> is a library that has commands for input from the keyboard and output to the screen.
+
+- Line 3 - **using namespace std;**
+
+It is something that you cannot avoid and have to write in every single program that you will write in your whole life.
+
+- Line 4 - **int main ( ) {**
+
+The aforementioned tells the compiler that this is the beginning of the main program. **{** means the start of the block of the program.
+
+- Line 5 - **cout<<"Hello World"<<endl;**
+
+**cout** is our "output to the screen" statement. **"endl"** takes you the next line. **";"** ends a C++ statement.
+
+- Line 6 - **return 0**
+
+This means that this is the end of our program.
+
+- Line 7 - **}**
+
+The aforementioned is our closing bracket to end the program.
+
+Congratulations! You just wrote your first C++ program.
+
+Let's write another program that would read two numbers and output the addition of them.
+
+`   
+     // program to add two numbers
```

Comment:
![image](https://user-images.githubusercontent.com/60800358/89741338-a98abc00-da4d-11ea-9be7-8e5697397535.png)

@param31  - not sure if this code block is supposed to render like this?

Reply:
Yeah, this isn't how it is supposed to be. I formatted it again. Just the way how it should be.